install-base.sh

```diff
@@ -33,7 +33,8 @@ fi
 
 #### Install FCM dependencies & configuration
 if [[ $dist == ubuntu ]]; then
-  apt-get install -q -y subversion firefox tkcvs tk libxml-parser-perl || error
+  apt-get install -q -y subversion chromium-browser tkcvs tk libxml-parser-perl || error
+  xdg-settings set default-web-browser chromium-browser.desktop
```

Comment:
The Cylc 8 graph view seems to have weird display issues with Firefox on the Ubuntu 22.04 VM (never seen this issue with Firefox anywhere else). Switching to Chromium for the moment.